{"uid":"a0b6e13b60cbd7f3","name":"Board Deletion","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_board_deletion","historyId":"4bab3c1ca3a9b77bdca99b9a1a8bf44f","time":{"start":1734050983538,"stop":1734051016198,"duration":32660},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f68528e1820>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"90da1b98ba0c4649a3167cfb1e37f3d1\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"90da1b98ba0c4649a3167cfb1e37f3d1\")>\nmethod = <function url_matches.<locals>._predicate at 0x7f685185cf70>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f68528e1820>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"90da1b98ba0c4649a3167cfb1e37f3d1\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC009\n    @allure.title('Board Deletion')\n    def test_board_deletion(self, driver):\n        time.sleep(5)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:263: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:2082: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1589: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1599: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1661: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:954: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1093: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1085: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:929: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:668: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:373: TypeError","flaky":true,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1734050983258,"stop":1734050983537,"duration":279},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1734050983537,"stop":1734050983537,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f68528e1820>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"90da1b98ba0c4649a3167cfb1e37f3d1\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"90da1b98ba0c4649a3167cfb1e37f3d1\")>\nmethod = <function url_matches.<locals>._predicate at 0x7f685185cf70>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f68528e1820>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"90da1b98ba0c4649a3167cfb1e37f3d1\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC009\n    @allure.title('Board Deletion')\n    def test_board_deletion(self, driver):\n        time.sleep(5)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:263: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:2082: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1589: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1599: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1661: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:954: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1093: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1085: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:929: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:668: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:373: TypeError","steps":[{"name":"Log in to Trello account","time":{"start":1734050988540,"stop":1734051016198,"duration":27658},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 263, in test_board_deletion\n    self.test_login(driver)\n  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 2082, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1446, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1589, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1599, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1661, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 954, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1093, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1085, in emit\n    msg = self.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 929, in format\n    return fmt.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 668, in format\n    record.message = record.getMessage()\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 373, in getMessage\n    msg = msg % self.args\n","steps":[{"name":"Navigate to the Trello login page","time":{"start":1734050988540,"stop":1734050991925,"duration":3385},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1734050991925,"stop":1734051016139,"duration":24214},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 2082, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1446, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1589, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1599, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1661, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 954, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1093, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1085, in emit\n    msg = self.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 929, in format\n    return fmt.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 668, in format\n    record.message = record.getMessage()\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 373, in getMessage\n    msg = msg % self.args\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":2,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"76530eeddaf127d","name":"log","source":"76530eeddaf127d.txt","type":"text/plain","size":63294}],"parameters":[],"stepsCount":3,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1734051016367,"stop":1734051016438,"duration":71},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC009"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1778-882"},{"name":"thread","value":"2819-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":2,"skipped":0,"passed":7,"unknown":0,"total":10},"items":[{"uid":"1f3664d9e40b15f1","status":"passed","time":{"start":1734027580701,"stop":1734027614323,"duration":33622}},{"uid":"706f5fe60a69aac4","status":"passed","time":{"start":1733973950982,"stop":1733973984641,"duration":33659}},{"uid":"d92c1d7ce43d52ba","status":"failed","statusDetails":"AssertionError: assert (3 - 2) == 0","time":{"start":1733972817253,"stop":1733972850983,"duration":33730}},{"uid":"126019420d6637e0","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733971848683,"stop":1733971878235,"duration":29552}},{"uid":"d2c8982d216e2b41","status":"passed","time":{"start":1733970594572,"stop":1733970624746,"duration":30174}},{"uid":"2f253d353a7ac6ed","status":"passed","time":{"start":1733963472621,"stop":1733963503631,"duration":31010}},{"uid":"2b935169c8b21eab","status":"passed","time":{"start":1733932745107,"stop":1733932775928,"duration":30821}},{"uid":"b395601e78c5e29","status":"passed","time":{"start":1733891584210,"stop":1733891614926,"duration":30716}},{"uid":"6eda68015350a44e","status":"passed","time":{"start":1733889978687,"stop":1733890009499,"duration":30812}}]},"tags":["TC009","TC000"]},"source":"a0b6e13b60cbd7f3.json","parameterValues":[]}